mod/questionnaire/mobile_css.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
Not sure what this file is. Can you explain?

Reply:
Sorry for confusing you. In our project, we want to allow CSS files can be accessed from external requests, so we add some headers.
This is specific to our project so we will revert this change.